qiskit/providers/honeywell/honeywelljob.py

```diff
@@ -258,6 +258,13 @@ async def status(self, job_id, timeout=300):
                     await websocket.send(json.dumps(body))
                     api_response = await asyncio.wait_for(websocket.recv(), timeout=timeout)
                     api_response = json.loads(api_response)
+            else:
+                print('Websockets via proxy not supported.  Falling-back to polling.')
```

Comment:
Instead of using a `print` here is there a reason not to use the logger?

Reply:
No good reason.  :) I'll redirect it to the logger.